platform/platform_tests/distance_tests.cpp

```diff
@@ -89,17 +92,17 @@ UNIT_TEST(Distance_CreateAltitudeFormatted)
     Distance d = Distance::CreateAltitudeFormatted(12345);
     TEST_EQUAL(d.GetUnits(), Distance::Units::Meters, ());
     TEST_ALMOST_EQUAL_ULPS(d.GetDistance(), 12345.0, ());
-    TEST_EQUAL(d.GetDistanceString(), "12,345", ());
-    TEST_EQUAL(d.ToString(), MakeDistanceStr("12,345", "m"), ());
+    TEST_EQUAL(d.GetDistanceString(), "12" + groupSep + "345", ());
```

Comment:
Is there an option to set/override the locale from the test, so we can check different separators for different locales explicitly?

Reply:
Hi @biodranik: I'm afraid that with the current implementation, we may not be able to test the `Distance::FormatAltitude()` function for different locales.

The main reason is that this function calls the `measurement_utils::ToStringPrecision()` function, which uses a static variable to store the locale to use. This locale is therefore updated only in the first iteration, so if we change the locale during the tests, it won't have any effect.

We could change the behavior of the `measurement_utils::ToStringPrecision()` function by removing this static variable, but that would lead to a worse runtime performance.

Reply:
@biodranik , @vng: PTAL